plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergConnectorSmokeTest.java

```diff
@@ -112,6 +110,6 @@ protected void deleteDirectory(String location)
     @Override
     protected boolean isFileSorted(Location path, String sortColumnName)
     {
-        return checkOrcFileSorting(HDFS_FILE_SYSTEM_FACTORY.create(SESSION), path, sortColumnName);
+        return checkOrcFileSorting(fileSystem, path, sortColumnName);
```

Comment:
shows up in `Extract actual file system from Iceberg connector in tests` but probably belongs to `Cleanup file system usages in Iceberg tests`?